pkg/utils/k8sutil.go

```diff
@@ -819,8 +819,40 @@ func EnsureFuncDeployment(client kubernetes.Interface, funcObj *spec.Function, o
 	return err
 }
 
+func doRESTReq(restIface rest.Interface, groupVersion, verb, resource, elem, namespace string, body *[]byte) ([]byte, error) {
+	var req *rest.Request
+	switch verb {
+	case "get":
+		req = restIface.Get().Name(elem)
+		break
+	case "create":
+		req = restIface.Post().Body(*body)
+		break
+	case "update":
+		req = restIface.Put().Name(elem).Body(*body)
+		break
+	default:
+		return []byte{}, fmt.Errorf("Verb %s not supported", verb)
+	}
+	return req.AbsPath("apis", groupVersion, "namespaces", namespace, resource).DoRaw()
+}
+
+func doRESTCronJobReq(restIface rest.Interface, groupVersion, verb, elem, namespace string, body *[]byte) (batchv2alpha1.CronJob, error) {
+	jsonCronJob, err := doRESTReq(restIface, groupVersion, verb, "cronjobs", elem, namespace, body)
+	if err != nil {
+		return batchv2alpha1.CronJob{}, err
+	}
+	res := batchv2alpha1.CronJob{}
+	// We cannot use Into method to avoid the schema validation
```

Comment:
just curious about this comment: can you explain ?

Reply:
yes, it is not very clear now. What I wanted to say is that we cannot do `req.Do().Into(&cronjob)` since under the hood, client-go does a validation and if the group version is not expected (like with `v1beta1`) it throws an error. Instead of that we need to do manually `req.Do().Raw() => Unmarshal`